src/Conversion/ONNXToKrnl/ConvertONNXToKrnl.cpp

```diff
@@ -49,6 +80,10 @@ struct FrontendToKrnlLoweringPass
 void FrontendToKrnlLoweringPass::runOnOperation() {
   ModuleOp module = getOperation();
 
+  // Create an entry for this module
```

Comment:
This map seems to be important, can you add one or two sentences on what it contains, how it is used. It appears to be key to decide if an alloc can be moved or not.